helix-core/src/test/java/org/apache/helix/integration/rebalancer/WagedRebalancer/TestWagedRebalance.java

```diff
@@ -209,7 +229,7 @@ public void testRebalanceTool() throws InterruptedException {
     }
 
     utilResult = HelixUtil
-        .getIdealAssignmentForWagedFullAuto(ZK_ADDR, clusterConfig, instanceConfigs, liveInstances,
+        .getTargetAssignmentForWagedFullAuto(ZK_ADDR, clusterConfig, instanceConfigs, liveInstances,
```

Comment:
Let's add this section of test for the getImmediateAssignmentForWagedFullAuto too.

Reply:
Done so.